### PR TITLE
Refactor CSS files

### PR DESCRIPTION
**Pull request by tiffperumpail**
Make our stylesheets cleaner by taking CSS rules out of addPage.scss and creating new files.

### CODE REVIEW

File: app/assets/stylesheets/header.scss [hunk 1]
```diff
@@ -0,0 +1,67 @@
+@import "themeColors";
+
+#logo {
```
Comment (ErikOstien) [inline app/assets/stylesheets/header.scss]: This css was removed and is not used anymore. Was the PR rebased with master? This is not present currently in master. Don't know if other css that was removed was added again

Comment (ErikOstien) [reply app/assets/stylesheets/header.scss]: The #logo part

Comment (tiffperumpail) [reply app/assets/stylesheets/header.scss]: Removed #logo. Checked the rest of the rules and they exist on master.

**Review by ErikOstien — APPROVED** [review]
Removal of the #logo css looks good. Everything else was checked and this is just a moving around of files now. Looks good to merge.

**Pull request merged**